roles/confluent.test/molecule/rbac-mtls-rhel/verify.yml

```diff
@@ -4,6 +4,7 @@
 ### Validates that secrets protection is masking the correct properties.
```

Comment:
we also what to update the molecule.yml file to enable a `ssl.principal.mapping.rule` and verify that kafka starts up with it

Reply:
Added the rule and molecule file and added checks for property validation of mapping rule and super user. The cluster comes up properly and all checks are good. 

```
PLAY RECAP *********************************************************************
control-center1            : ok=59   changed=8    unreachable=0    failed=0    skipped=38   rescued=0    ignored=0
kafka-broker1              : ok=94   changed=13   unreachable=0    failed=0    skipped=59   rescued=0    ignored=0
kafka-broker2              : ok=78   changed=13   unreachable=0    failed=0    skipped=44   rescued=0    ignored=0
kafka-broker3              : ok=78   changed=13   unreachable=0    failed=0    skipped=44   rescued=0    ignored=0
kafka-connect1             : ok=67   changed=8    unreachable=0    failed=0    skipped=50   rescued=0    ignored=0
kafka-rest1                : ok=60   changed=10   unreachable=0    failed=0    skipped=37   rescued=0    ignored=0
ksql1                      : ok=64   changed=8    unreachable=0    failed=0    skipped=46   rescued=0    ignored=0
schema-registry1           : ok=59   changed=7    unreachable=0    failed=0    skipped=44   rescued=0    ignored=0
zookeeper1                 : ok=49   changed=6    unreachable=0    failed=0    skipped=42   rescued=0    ignored=0
```

Verify
```
PLAY RECAP *********************************************************************
control-center1            : ok=6    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
kafka-broker1              : ok=18   changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
kafka-broker2              : ok=18   changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
kafka-broker3              : ok=18   changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
kafka-connect1             : ok=14   changed=4    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
kafka-rest1                : ok=6    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
ksql1                      : ok=6    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
localhost                  : ok=4    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
schema-registry1           : ok=12   changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
zookeeper1                 : ok=2    changed=0    unreachable=0    failed=0    skipped=0    resc
```